modules/ossm-rn-known-issues.adoc

```diff
@@ -46,6 +44,11 @@ Workaround: Use the `techPreview` `jwksResolverExtraRootCA` field to enable addi
 +
 After enabling the `spec.security.controlPlane.mtls` setting in the SMCP, the Kiali console displays the following error message `No subsets defined`.
 
+* https://issues.redhat.com/browse/OSSM-1505[OSSM-1505] `ServiceMeshExtension` fails to be ready. If you are using the `ServiceMeshExtension` resource on OpenShift Container Platform 4.11 the resource never becomes ready. If you inspect the issue using `oc describe ServiceMeshExtension` you will see the following error:  `stderr: Error creating mount namespace before pivot: function not implemented`.
```

Comment:
Could we revise this to make it clearer that this is **_only_** a problem if you're on OCP 4.11?

Reply:
@JStickler I've updated the PR.